documentation/samplefiles/readme.go.md

```diff
@@ -5,42 +5,22 @@ These settings apply only when `--go` is specified on the command line.
 ```yaml $(go)
 go:
   license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: [[ServiceName]]
   clear-output-folder: true
 ```
 
 ### Go multi-api
 
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: package-2019-12-01
-  - tag: package-2020-07-01-preview
-  # add every tag listed below
+  - tag: package-[[Version]][[-ReleaseState]]
```

Comment:
Is this dash in the bracket kind of a conditional template string? Does this `[[-ReleaseState]]` mean the following?
```
result = ReleaseState ? `-${ReleaseState}` : ""
```

Reply:
as you said, it is~